server/lib/nl/fulfillment/fulfiller.py

```diff
@@ -131,6 +134,9 @@ def fulfill(uttr: Utterance) -> PopulateState:
   if params.is_special_dc(state.uttr.insight_ctx):
     _prune_non_country_special_dc_vars(state)
 
+  if state.uttr.mode == QueryMode.TOOLFORMER_RIG:
+    _prune_topic_vars_for_rig(state)
```

Comment:
Do this for non-multi var case only?

Reply:
sorry, not sure what you want me to check here .. there's nothing about multivar or not?

Reply:
just coz, we don't want to arbitrarily prune only LHS or RHS?  for multi-var its always 2 svs

Reply:
i this this should be handled with the new logic

Reply:
Hmmm, multi-var uses MultiSV and its very different.  So I'm not sure if this type of handling even makes sense?

Reply:
updated as we discussed offline